app/components/works/edit/attached_file_component.html.erb

```diff
@@ -0,0 +1,22 @@
+<tr class="attached-files <%= 'dz-complete' if uploaded? %>" data-dropzone-target="preview">
+    <%= form.fields_for :attached_files, attached_file do |file_form| %>
+    <td class="file-level" style="--depth: <%= depth+1 %>rem" data-dropzone-target="fileName">
+        <span class="fa-regular fa-file" aria-hidden="true"></span> <%= basename %>
```

Comment:
The filename currently looks like it's not vertically centered (like the hide checkbox) unlike the other elements in the row. Worth tossing that CSS around here?

Reply:
Yup, will align vertically.